Data_Structures_Questions.md

```diff
@@ -3,18 +3,20 @@ Answer the following questions for each of the data structures you implemented a
 ## Queue
 
 1. What is the runtime complexity of `enqueue`?
-
+   O(1)
 2. What is the runtime complexity of `dequeue`?
-
+   O(n)
```

Comment:
Good job with this one.  Most people assume removing from front of an array is just as easy as the back, but we have to shift over every element to make room when popping/appending to the front.

This is the reason a LinkedLIst is better to use for the Queue since operation on the head/tail is a O(1) operation.